administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -152,7 +158,12 @@ const ApplicationCard = ({
           )}
         </div>
       }
-      rightElement={<VerificationsQuickIndicator verifications={application.verifications} />}
+      rightElement={
```

Comment:
I think it would make sense to introduce a new component
"JulicaVerificationQuickIndicator".
Then here you can render the correct component depending on "isJuleicaEntitlementType()"